deploy/Makefile

```
@@ -0,0 +1,75 @@
+	kubectl -n $(NAMESPACE) port-forward svc/$(RELEASE) $(PORT):$(PORT) >/tmp/omni-pf.log 2>&1 & \
+	PF_PID=$$!; sleep 3; \
+	echo "GET /health:"; \
+	curl -fsS localhost:$(PORT)/health || echo "  (not healthy yet)"; \
+	echo; kill $$PF_PID 2>/dev/null || true
+
+.PHONY: uninstall
+uninstall: ## Remove the release
+	helm uninstall $(RELEASE) --namespace $(NAMESPACE) || true
+
+.PHONY: smoke
+smoke: ## Run the image locally (expects DB-connect wait without a DB)
```

checkmake (0.3.2)

[warning] Required target "all" is missing from the Makefile.

(minphony)